app/models/entry.server.ts

```diff
@@ -50,6 +50,29 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export function getEntriesByInitialLettersAndPage(
+  initialLetters: string,
+  page: string
+) {
+  const pageNumber: number = parseInt(page);
+  if (isNaN(pageNumber)) {
+    throw new Error(
+      `Page Number ("${page}") must be a number`
+    )
+  } else if (isNonPositive(pageNumber)) {
+    throw new Error(
+      `Page Number ("${page}") must be greater than zero`
+    )
+  }
+
+  const skip: number = (pageNumber - 1) * 100;
```

Comment:
This is relying on the `take` argument to `getEntriesByInitialLetters` having a default of `100`. Let's factor out this value into a constant at the top of the file:

```ts
const DEFAULT_PAGE_SIZE = 100
```
And then use that constant here, and down below in the function declaration for `getEntriesByInitialLetters`.

Reply:
Done